charm/src/iframe/static.ts

```diff
@@ -596,12 +631,8 @@ async function fetchLLMResponse() {
 \`\`\`jsx
 async function fetchFromUrl() {
   const url = 'https://twopm.studio';
-  try {
-    const result = await readWebpage(url);
-    console.log('Markdown:', result.content);
-  } catch (error) {
-    console.error('readWebpage error:', error);
-  }
+  const result = await readWebpage(url);
+  console.log('Markdown:', result.content);
```

Comment:
Prefer to throw rather than suppress errors so we can catch them at a higher layer.